Objects/longobject.c

```diff
@@ -41,6 +41,9 @@ PyObject *_PyLong_One = NULL;
    -NSMALLNEGINTS (inclusive) to NSMALLPOSINTS (not inclusive).
 */
 static PyLongObject small_ints[NSMALLNEGINTS + NSMALLPOSINTS];
+
+#define IS_SMALL_INT(ival) (-NSMALLNEGINTS <= (ival) && (ival) < NSMALLPOSINTS)
```

Comment:
Why not make this a function?

Reply:
I actually have a patch for that too :slightly_smiling_face: : https://github.com/gnprice/cpython/commit/cfc91e6ca

I left that out of this commit in order to change the fewest possible aspects at once, and figured I'd send it as a separate followup provided this one is merged. But I'd also be happy to add it to this PR, if there's agreement that it's desirable; I think it'd fit well as part of the same change.

(Note the "before" side of that commit looks slightly different from the "after" of this one, as there are a couple of other commits in between on that development branch. But it'd be no trouble for me to rebase it to directly on top of this one in order to add to this PR.)

Reply:
Okay, if making a function is incoming, this PR lgtm.

I'll let Raymond weight in.

Reply:
Go ahead and make it a static inlined function.

Reply:
This is a somewhat minor change, but let's add a blurb NEWS entry in case anyone cases about these details.

Reply:
Sounds good! Thanks for the review.